Sources/VaporToolbox/Clean.swift

```diff
@@ -132,9 +123,7 @@ class Cleaner {
     }
 
     private func cleanDefaultDerivedDataLocation() throws -> Bool {
-        let defaultLocation = try Process.shell.homeDirectory()
-            .trailingSlash
-            + "Library/Developer/Xcode/DerivedData"
+        let defaultLocation = try FileManager.default.url(for: .libraryDirectory, in: .userDomainMask, appropriateFor: nil, create: false).path.appendingPathComponents("Developer", "Xcode", "DerivedData")
```

Comment:
We should also search for relative derived data in the current working directory as well

Reply:
We do, check a few lines down.

Reply:
🤦‍♂